react/index.rst

```diff
@@ -0,0 +1,35 @@
+.. _react-label:
+
+=====
+React
+=====
+
+This is the documentation for the "React" training.
+
+The React training is intended as a single day training for people who are new to React but have knowledge of JavaScript and ES6.
+
```

Comment:
please see other index sites how we are doing that, I miss here the description, the about and other settings :)